### ISSUE TITLE
CRUD does not like a column called 'fat'

### ISSUE THREAD

**Issue by rabol** [Laravel-Backpack/CRUD#2162]
# Bug report

### What I did
Created a new migration
php artisan make:migration create_tests_table
php artisan backpack:crud test

in the newly created migration file:

add this:

$table->string('fat')->nullable();

migrate the database
open the admin panel and select the new Tests
and the list will only show 'F' and when adding a new record, the label i 'F'

### What I expected to happen
I would expect the label in the list to be 'Fat' and when creating a new record the label should be 'Fat'

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:

PHP 7.3.9-1+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Sep  2 2019 12:54:24) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.9, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.9-1+ubuntu18.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies

### LARAVEL VERSION:
laravel/framework                     v6.2.0     The Laravel Framework.

### BACKPACK VERSION:
backpack/crud                         4.0.8      Quickly build an admin interfaces using Laravel 6, CoreUI, Boostrap 4 and jQuery.
backpack/generators                   2.0.4      Generate files for laravel projects

**Comment by rabol** [comment]
Acutally the same happends if the column is name Cat, Xat, so it looks like the CRUD does not like 3 letter columns that ends in 'at'

**Comment by jsvini** [comment]
@rabol this happens because AutoSet trait removes `at|id` from the end of the column **label**:

https://github.com/Laravel-Backpack/CRUD/blob/55ecff99dd92823e7697a717b2e2adca12e45f08/src/app/Library/CrudPanel/Traits/AutoSet.php#L193

I think that's because of default eloquent columns like created_at, updated_at, deleted_at, or relations columns like client_id, book_id and so on. 

You can change the automatically generated column label:

```php
$this->crud->setFromDb();
$this->crud->modifyColumn('fat', ['label' => 'Fat']);
```

**Comment by johnpuddephatt** [comment]
Shouldn't AutoSet only be matching `_id` and `_at` not `id` and `at` ?

**Comment by tabacitu** [comment]
Totally agree! We should remove "_id" and "_at", not "id" and "at". Done! Will be available with a ```composer update``` as 4.0.10. Thank you for tracking it down @jsvini , and @johnpuddephatt for the solution.

@rabol to answer your question - yes, you should be able to have a column called something with ID. And you can, by defining the label yourself. But in 99% of the cases, I think the dev would prefer to have the _id or _at stripped - so I think that should be the default, the 99% use case. Again, this is only something that ```autoSet``` does - and we discourage using ```autoSet``` in most cases. For most CRUDs you should manually define columns and fields.
